conda/recipes/rapids-build-env/meta.yaml

```diff
@@ -62,6 +62,8 @@ requirements:
     - distributed {{ distributed_version }}
     - dlpack {{ dlpack_version }}
     - double-conversion {{ double_conversion_version }}
+    - faiss-proc=*=cuda {{ faiss_version }}
```

Comment:
Typically these `proc` packages just use version `1.0.0`so you don't want to version them

Reply:
I've removed the faiss version